### DISCUSSION TITLE
AAA-auth mobile

### DISCUSSION THREAD

**Discussion by testanykey12345** [emanuele-f/PCAPdroid#427] [Q&A] [answered]
Has PCAPdroid AAA-auth cellular network packet capture support? For example, RADIUS or DIAMETER

**Comment by emanuele-f** [comment, marked as answer]
Hello,
PCAPdroid only captures the traffic of the device where it's installed, afaik normally these protocols are used in the infrastructure. So this question is more a traffic visibility question, if PCAPdroid has visibility on it then it will capture it

**Reply by testanykey12345** [reply]
Hello,
Thank you for information
